msal/src/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -39,12 +39,6 @@ internal class SilentRequest : RequestBase
         public SilentRequest(AuthenticationRequestParameters authenticationRequestParameters, bool forceRefresh)
             : base(authenticationRequestParameters)
         {
-            if (authenticationRequestParameters.Account == null)
-            {
-                throw new MsalUiRequiredException(MsalUiRequiredException.UserNullError, "Null user was passed in AcquiretokenSilent API. Pass in " +
```

Comment:
You can add the error message to a constant file and it will be easier to test for it as well.

Reply:
yes...will do.

Reply:
done.